src/components/editor/editor-window/key-map.ts

```diff
@@ -0,0 +1,109 @@
+import CodeMirror, { KeyMap } from 'codemirror'
+import { wrapTextWith } from './tool-bar/utils'
+
+// const isMac = CodeMirror.keyMap.default === CodeMirror.keyMap.macDefault
```

Comment:
Is this needed or can this safely be deleted?

Reply:
It was needed in the old code, but it's so stupid that I removed it, but I think we may need to readd this. Maybe we should rework the shortcuts altogether and remove the old ones for now?

Reply:
Please let's try to build a clean solution

Reply:
What was its purpose?

Reply:
it was used on some shortcuts to differentiate between macs and non-macs, but the latest changes should have made that obsolete